src/main/java/uk/ac/cam/cl/dtg/segue/api/UsersFacade.java

```diff
@@ -344,22 +344,18 @@ public Response generatePasswordResetTokenForOtherUser(@Context final Request re
                                     LOCAL_AUTH_EMAIL_FIELDNAME, userOfInterest.getEmail(),
                                     LOCAL_AUTH_GROUP_MANAGER_EMAIL_FIELDNAME, currentUser.getEmail(),
                                     LOCAL_AUTH_GROUP_MANAGER_INITIATED_FIELDNAME, true));
-            return Response.ok().build();
 
+            log.info("Password reset requested for account: " + userIdOfInterest);
+            return Response.ok().build();
         } catch (NoUserException e) {
-            log.warn("Password reset requested for account that does not exist: " + e.getMessage());
+            log.warn("Password reset requested for account that does not exist: " + userIdOfInterest);
```

Comment:
does this user id need sanitising?

Reply:
While this is provided by the request, it's typed as a Long. Do we need to manually restrict that or would Java just error on a non-numeric input as-is?

Reply:
that's fine. If anything that can't be coerced into a long appears, it'll throw an exception and be caught by the handler.

Reply:
Assumed that would be the case but good to have that confirmed... 👍